model/webhook.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+const (
+	// TypeAccount is the string value that represents an account
+	TypeAccount = "account"
+)
+
+// Webhook is
```

Comment:
I very much want to know what this webhook is. :D

Reply:
Hahahaha I always love it when I miss those things :P